.sonarcloud.properties

```
@@ -1,3 +1,5 @@
+sonar.organization=airsidemobile
+
```

Should we have it notify github of the results as well?
```suggestion
sonar.pullrequest.github.repository=airsidemobile/JOSESwift
```